lib/ISAACConfig.cmake

```diff
@@ -64,7 +64,7 @@ if (ISAAC_JPEG)
         endif()
 endif (ISAAC_JPEG)
 
-set(ISAAC_VECTOR_ELEM "2" CACHE STRING "The amounts of elements used for vectorization. On GPU 1..2 should be fine, on CPU 4..16" )
+set(ISAAC_VECTOR_ELEM "1" CACHE STRING "The amounts of elements used for vectorization. On GPU 1 should be fine, on CPU 4..32, depending on the vectorization capabilities" )
```

Comment:
Tested on a Intel(R) Core(TM) i5-4200M with gcc 4.9.2. I think the GCC version is more important than the CPU generation. I will try on laser later, but first I need to get it built. ;)